index.html

```diff
@@ -4,7 +4,7 @@
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <script defer src="fizzbuzz.js"></script>
+        <script defer src="8_1_Retry.js"></script>
```

Comment:
you could use `<script type="module" src="/index.js"/>`  that imports `index.js` which is exporting other modules